typing/typemode.ml

```diff
@@ -462,11 +462,10 @@ let untransl_modality (a : Modality.t) : Parsetree.modality loc =
 
 (* For now, mutable implies legacy modalities for both comonadic axes and
    monadic axes. In the future, implications on the comonadic axes will be
-   removed (and can be experimented currently with using
-   @no_mutable_implied_modalities). The implications on the monadic axes will
-   stay. *)
+   removed. The implications on the monadic axes will stay. Implied modalities
```

Comment:
Is this still the plan (to remove the implied modalities)? I suppose this patch makes it no harder to do that, and it simplifies things in the meantime. Just checking to make sure this comment is still accurate; just Resolve this if it is.

Reply:
Yeah it's still our plan to remove the implied modalities and this PR doesn't make that harder.